docs/linux/reporting_kernel_bugs.md

```diff
@@ -52,61 +52,119 @@ If you can't figure out the right fix, but have some understanding of the bug, p
+There are no guarantees that the fix will be propagated to stable or distro kernels.
+The maximum embargo on this list is **7 days from the moment** from the start of the released fix process,
+whereas for "linux-distros" the embargoed period starts from the initial post to the list
+regardless of the availability of a fix.
+
+That's why it's strongly recommended, do NOT include "[linux-distros](https://oss-security.openwall.org/wiki/mailing-lists/distros)" mailing lists in CC
+UNTIL the fix is accepted by affected systems' maintainers.
+>  In other words, until a fix is accepted do not Cc: “linux-distros”, and after it’s merged do not Cc: the kernel security team.
+
+During reporting the security bug to `security@kernel.org`, keep everything in **plain text**, including the fix, exploit code etc.,
+think of it like a [regular Linux kernel patch submission](https://docs.kernel.org/process/submitting-patches.html).
+
+Note that `security@kernel.org` doesn't assign CVE as they simply do not require it. See CVE assignment instructions below.
 
-A few notes:
+### CVE assignment
 
-* There should ideally be no delay between reports to `security@kernel.org` and `linux-distros@vs.openwall.org`.
-* When working on the patch together with the `security@kernel.org` members and upstream maintainers, keep the linux-distros aware of the progress.
-* There should ideally be no delay between CVE description publication, distros' updates, upstream commit and notification to `oss-security@lists.openwall.com`. All of these should be on the same day, at worst.
-* The moment the issue is made public (e.g. patch is submitted upstream, CVE description published, etc.) it must be reported to `oss-security@lists.openwall.com` right away.
+kernel.org is now CNA (from [February 13, 2024](https://www.cve.org/Media/News/item/news/2024/02/13/kernel-org-Added-as-CNA)),
+therefore they have reserved CVE IDs for the kernel vulnerability bugs classified as CVE.
 
-A good example of an LPE announcement structure on `oss-security@lists.openwall.com` can be found [here](http://seclists.org/oss-sec/2016/q4/607), however the timeline doesn't look right there: public announcement should have occurred right after the patch was submitted to netdev.
+CVE assignments (sometimes rejections) are publicly announced in public list, see [linux-cve-announce](https://lore.kernel.org/linux-cve-announce) on which you can [subscribe](https://subspace.kernel.org/subscribing.html).
+
+CVEs are assigned after-the-fact with 1-2 week delay usually.
+> No hardware CVEs (SPECTRE-like CVEs come from others) (c) Greg K-H
+
+, e.g. bugs in drivers are Linux kernel related, hence they can be treated as CVEs, and HW bugs in CPU, NIC, other chips that affect Linux kernel, Windows, OSX etc.
+that are not caused due to Linux kernel code, should be handled by HW vendor.
+
+Averaging 55 CVEs per week (2024 Q3):
+> CVE assignment team is overly cautious and assign CVE numbers to any bugfix that they identify
```

Comment:
"The CVE assignment team is overly cautious and assigns..."
Also, please add a period at the end of the sentence.

Reply:
I also think we can go without "overly", as this sorta diminishes their efforts.

How about: "The CVE assignment team is very thorough and assigns CVE numbers to any bugfix that they identify."